src/main/webapp/WEB-INF/tags/module/ar/invoiceGeneral.tag

```diff
@@ -24,7 +24,9 @@
 <c:set var="invoiceAccountDetailsAttributes" value="${DataDictionary.InvoiceAccountDetail.attributes}" />
 <c:set var="invoiceGeneralDetailAttributes" value="${DataDictionary.InvoiceGeneralDetail.attributes}" />
 <c:set var="documentAttributes" value="${DataDictionary.ContractsGrantsInvoiceDocument.attributes}" />
-<c:set var="readOnlyForFinal" value="${readOnly || not KualiForm.document.finalizable}" />
+<c:set var="editFinalBill"
```

Comment:
Do you think this file should mention which sections were backported?

Reply:
comment added